### DISCUSSION TITLE
bind sso user to local user

### DISCUSSION THREAD

**Discussion by Julie4953** [netbox-community/netbox#17904] [Help Wanted!] [answered]
HI, I installed Netbox 4.1.5 and imported the dump of version 3.4.10 via 3.7.10.
I got all my information back. I wanted to set up my SSO (Keycloak) it connects me well but it creates the users during the connection with a character string behind because it already exists in the postgresql database except that I want it to associate the keycloak SSO account to netbox account.
If anyone has ever had a problem or knows how to fix it, I'd appreciate it.

**Comment by avbor** [comment]
Yeap, same question. Is any way to merge local user with sso-user, or, add sso for local user?

**Reply by avbor** [reply, marked as answer]
I found solution here - https://www.oasys.net/posts/okta-sso-with-netbox/
With SOCIAL_AUTH_PIPELINE it's possible to merge local user with sso.

**Reply by Julie4953** [reply]
Thanks a lot. I put the assiciate by email and it works !
